articles/zcash-protocol/chapter_3.md

```diff
@@ -0,0 +1,209 @@
+---
+Title: Concepts
+Status: Done
+Level: "3"
+---
+
+# Chapter 3: Concepts
+## 3.1 Payment addresses and keys
+---
+Users who wish to receive shielded payments in the $\texttt{Zcash}$ protocol must have a shielded payment address generated from a spending key.
+
+![](../../articles/zcash-protocol/attachments/shielded_payment_addresses.png)
+
+- The diagram compares the differences between key components in Sprout, Sapling, and Orchard.
```

Comment:
You should create a reference to the diagram so that readers know which diagram you are describing.